Documentation/gettingstarted/istio.rst

```diff
@@ -22,6 +22,15 @@ environment running on your machine.
    instructions provided here for the other GSGs. 5 GB and 4 CPUs
    should be enough for this GSG (``--memory=5120 --cpus=4``).
 
+.. note::
+
+   If Cilium is deployed in kube-proxy-free mode, you need to set
+   ``--bpf-socket-lb-hostns-only: true`` in the deployment yaml
+   directly or via ``bpf.socketHostnsOnly`` option with helm.
+   Without this option, Cilium will perform socket load balancing
```

Comment:
```suggestion
   Without this option, Cilium may translate Kubernetes service cluster IP for pod traffic using socket level load balancing
```

Reply:
curisou, is this a generally better tone or? my understanding is that in kube-proxy-free mode, Cilium *will* do socket lb for sure.

Reply:
In case of kube-proxy-free mode, socket load balancing is more efficient than the packet level load balancing that can happen at the tc layer. Some of the functionality is dependent on a minimum kernel version. How about this - "Without this option, when cilium translates..."?

Reply:
Done, PTAL :)